node/src/main/scala/com/wavesplatform/settings/WavesSettings.scala

```diff
@@ -60,4 +66,28 @@ object WavesSettings extends CustomValueReaders {
       rootConfig
     )
   }
+
+  def loadRootConfig(external: Option[File] = None): WavesSettings = {
```

Comment:
This is a bad design decision: reading and parsing settings has nothing to do with application component initialization.